.pre-commit-config.yaml

```diff
@@ -1,22 +1,12 @@
 repos:
 - repo: https://github.com/antonbabenko/pre-commit-terraform
-  rev: v1.92.0
+  rev: v1.89.1
   hooks:
     - id: terraform_fmt
       args:
         - --hook-config=--retry-once-with-cleanup=true
         - --args=-write=true
-        - --args=-recursive=true
-    - id: terraform_validate
-      args:
-        - --tf-init-args=-upgrade
-        - --hook-config=--retry-once-with-cleanup=true
-    - id: terrascan
-    - id: terraform_tfsec
-    - id: terraform_tflint
-      args:
-        - --args=--module
-        - --args=--enable-rule=terraform_documented_variables
+        - --args=-recursive=false # should be returned to true after solving issues with examples dependencies (vpc module in TF registry)
```

Comment:
why not back to true?

Reply:
Because this is being blocked by some issue which I am not ready to fix yet